packages/interactive_media_ads/ios/interactive_media_ads/Sources/interactive_media_ads/AdsManagerProxyAPIDelegate.swift

```diff
@@ -47,4 +47,12 @@ class AdsManagerProxyAPIDelegate: PigeonApiDelegateIMAAdsManager {
   func destroy(pigeonApi: PigeonApiIMAAdsManager, pigeonInstance: IMAAdsManager) throws {
     pigeonInstance.destroy()
   }
+
+  func adCuePoints(pigeonApi: PigeonApiIMAAdsManager, pigeonInstance: IMAAdsManager) throws
+    -> [Double]
+  {
+    return pigeonInstance.adCuePoints.map { cuePoint -> Double in
+      return (cuePoint as! NSNumber).doubleValue
```

Comment:
Is there an easy way for us to file an issue about the fact that this array should be an `NSArray<NSNumber*>` instead of an `NSArray`?

(Not to block this PR, of course, it's just weird that this is handled via documentation rather than the actual type system.)

Reply:
I will reach out to the team. I also need to followup on the issue with ui elements on iOS: https://github.com/flutter/packages/blob/main/packages/interactive_media_ads/ios/interactive_media_ads/Sources/interactive_media_ads/AdProxyAPIDelegate.swift#L41